### PR TITLE
components/appbar - rewrite dashboard appbar and layout

### PR DESCRIPTION
**Pull request by junhoyeo**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (softmarshmallow) [inline .prettierrc:1]: May i ask which convention this prettierrc follows?

File: web/sections/canvas-preview/index.tsx [hunk 2]
```diff
@@ -1,40 +1,46 @@
 import React, { useState } from "react";
```
Comment (softmarshmallow) [inline web/sections/canvas-preview/index.tsx:1]: @You-J  this part conflicts your working area. since @junhoyeo requests PR first, i'll accept this

**Review by softmarshmallow — APPROVED** [review]

File: .prettierrc (hunk 1 above)
Comment (junhoyeo) [reply .prettierrc:1]: I thought I needed to set a consistent convention for efficient work, so I first applied the rules that I thought was most visible in other files. The other prettierrc files I added to the other libs are mostly the same.
Someday we should discuss about the right rules and apply it throughout the project.

File: web/sections/canvas-preview/index.tsx (hunk 2 above)
Comment (junhoyeo) [reply web/sections/canvas-preview/index.tsx:1]: Please note that I just added the name for the default function for more efficient fast refreshing :fire:
The rest hasn’t changed much, except for the linting part lol

> Anonymous arrow functions cause Fast Refresh to not preserve local component state.

Comment (softmarshmallow) [reply web/sections/canvas-preview/index.tsx:1]: I understand

**Pull request merged**